fdbserver/ResolverInterface.h

```diff
@@ -77,10 +77,11 @@ struct ResolveTransactionBatchReply {
 	VectorRef<uint8_t> committed;
 	Optional<UID> debugID;
 	VectorRef<VectorRef<StateTransactionRef>> stateMutations;  // [version][transaction#] -> (committed, [mutation#])
+	std::map<int, Standalone<VectorRef<KeyRangeRef>>> conflictingKeyRangeMap; // transaction index -> conflicting keyRanges given by the resolver
```

Comment:
```suggestion
	std::map<int, VectorRef<KeyRangeRef>> conflictingKeyRangeMap; // transaction index -> conflicting keyRanges given by the resolver
```
I think this doesn't have to be standalone - you can use the Arena instead

Reply:
As we discussed before, I changed the code to use the index instead of keys.
I changed the data structure to `std::vector` here, which is easier to use.
Also, the `arena` in `ResolveTransactionBatchReply` is changed somewhere in the proxy code.
Since I have not found the place where the change happens, I just avoid using the `arena` here.